CreatePackagesQuickStart.md

```diff
@@ -30,7 +30,7 @@ Here's a TL;DR quick start version of the package creating tutorial. Follow thes
 * **Test the package**
    * **Testing should probably be done on a Virtual Machine**
    * In your package directory, use:
-      * `choco install package-name -s '$pwd' -f` - powershell
+      * `choco install package-name -s "$pwd" -f` - powershell
```

Comment:
I don't think quotes are necessary at all in this case. I don't think PowerShell does any expansion of the value, it passes it whole to the command, right?